app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -73,4 +74,36 @@ def supports_authentication?(authtype)
   def self.event_monitor_class
     ManageIQ::Providers::Openstack::InfraManager::EventCatcher
   end
+
+  def clouds
+    overclouds = ManageIQ::Providers::Openstack::CloudManager.find(:all, :conditions => ["id != ? and provider_id = ?", id, provider_id])
```

Comment:
We already have one of these: `provider.cloud_ems`. (Though I thought we'd renamed it to something more plural-sounding.)